openedx/core/djangoapps/notifications/tasks.py

```diff
@@ -78,15 +83,19 @@ def send_notifications(user_ids, course_key, app_name, notification_type, contex
     """
     Send notifications to the users.
     """
+    if not ENABLE_NOTIFICATIONS.is_enabled():
```

Comment:
We also need to pass the course id here.  We want to initially enable it for selected courses